theano/gpuarray/opt.py

```diff
@@ -2005,3 +2005,28 @@ def local_gpu_solve(op, context_name, inputs, outputs):
                                local_abstractconv3d_gradinputs_gemm, 30,
                                'conv_gemm',
                                'gpuarray', 'fast_compile', 'fast_run')
+
+
+# Register cuDNN batch normalization implementation
+abstract_batch_norm_groupopt = theano.gof.optdb.LocalGroupDB()
+abstract_batch_norm_groupopt.__name__ = "gpuarray_batchnorm_opts"
+register_opt('fast_compile')(abstract_batch_norm_groupopt)
```

Comment:
No specific reason for the LocalGroupDB, other than that this was how it is done for the convolution ops and I'm still trying to understand the different ways to register optimizations.